meson.build

```diff
@@ -162,8 +190,25 @@ threadlib = dependency('threads', required: true)
 # Optional dependency on GObject
 build_gobject = false
 if get_option('enable-gobject-types')
-  gobject = dependency('gobject-2.0', version: '>= 2.30.0', required: false)
-  build_gobject = gobject.found()
+  # MSVC builds typically do not use pkg-config, so find GObject/GLib manually
```

Comment:
In the long term I think depending on pkg-config on Windows is only a good thing. It isn't a massive dependency, is required by other layers of the stack (gobject-introspection), and generally makes building easier with build systems such as Meson.